_pages/author-shauryam.html

```diff
@@ -0,0 +1,28 @@
+---
+title: "Shauryam Jain"
+layout: default
+permalink: "/author-shauryam-jain"
```

Comment:
The value of permalink should match with the file name

## Changes to make
1. Either make the permalink to `permalink: "/author-shauryam"`
2. Or change the filename to from ` _pages/author-shauryam.html` to ` _pages/author-shauryam-jain.html`

Reply:
I actually tried "/author-shauryam" but h link was not working in the local server but after making it to shauryam-jain it worked.

Reply:
Let me reproduced this issue on my local system, but probably the fix that I told should work

Reply:
Let me see it, I am working on Dockerizing the project

Reply:
Any update on this or should we make a new PR.